api/datastore/postgres/postgres.go

```diff
@@ -410,14 +411,15 @@ func scanRoute(scanner rowScanner, route *models.Route) error {
 		return err
 	}
 
-	if headerStr == "" {
-		return models.ErrRoutesNotFound
+	if len(headerStr) > 0 {
+		json.Unmarshal([]byte(headerStr), &route.Headers)
 	}
 
-	if err := json.Unmarshal([]byte(headerStr), &route.Headers); err != nil {
-		return err
+	if len(configStr) > 0 {
+		json.Unmarshal([]byte(configStr), &route.Config)
 	}
```

Comment:
I understand bypassing these if they are nil/empty, but why ignore the returned errors?